### ISSUE TITLE
Cannot play only one video in main player (or audio in background) from a playlist

### ISSUE THREAD

**Issue by alittlebitofit** [TeamNewPipe/NewPipe#8499]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.23.0

### Steps to reproduce the bug

1. Go to any playlist with more than one video in it.
2. Longpress a video (not the last video) for popup menu.

### Expected behavior

I want to be able to play only one video either in main player or popup player or background.

I want that option in the popup menu when I long press the video in a playlist.

I know I can just tap on the video and click on Play or Play in Background, but thats annoying because first I would have to wait for the video to load in the Details screen and then Play.

Having the option in the popup menu saves that waiting period.

### Actual behavior

`Start playing in a popup` exists which plays the video and then the queue (which had only one video playing) ends.

But `Start playing in the background` adds the entire playlist to the queue.

Similarly, there's no way to `Start playing in main player` to play a single video and end the queue.

**Comment by opusforlife2** [comment]
This isn't a bug. It's a feature request. You're asking for functionality to be added.

**Comment by SameenAhnaf** [comment]
Duplicate of https://github.com/TeamNewPipe/NewPipe/issues/5741

**Comment by alittlebitofit** [comment]
> This isn't a bug. It's a feature request. You're asking for functionality to be added.

So sorry.

**Comment by alittlebitofit** [comment]
> Duplicate of #5741

@SameenAhnaf will that also solve for the background player?

**Comment by SameenAhnaf** [comment]
Yes. Currently, starting playback on main and background player work differently. But that solution should let the user get full control on how to proceed in this regard.

**Comment by opusforlife2** [comment]
I wouldn't say this is a duplicate. The solution _may_ end up being the same as the linked issue, but that is not guaranteed.

**Comment by SameenAhnaf** [comment]
@opusforlife2 Duplicate of https://github.com/TeamNewPipe/NewPipe/issues/7464

**Comment by opusforlife2** [comment]
Ah yes. It's a bit poorly explained, though, and the video link is expired, so I'll close that one instead.

**Comment by opusforlife2** [comment]
@tippylion28, regarding #6999, the OP seems to state the opposite of your issue. Can you confirm these steps once?

**Comment by TippyLion28** [comment]
@opusforlife2

I'm seeing inconsistent behaviour between play options. Playing in popup or main player just plays 1 video. Playing in the background causes the entire playlist to be added to the queue.

#6999 is essentially asking for the behaviour to be consistent, with the default being that the entire playlist gets queued, and playback begins at the chosen point in the playlist.

**Comment by eternal-sorrow** [comment]
This one is still a problem. There is a huge button "Background" at the top of the playlist interface. When I press on a single video in the playlist, I never mean to play the whole playlist, so this is very weird and unintuitive behaviour.
